packages/cst-dts-gen/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@chevrotain/cst-dts-gen",
+  "version": "9.0.0",
+  "description": "Generates type definitions for chevrotain CST nodes",
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/Chevrotain/chevrotain/issues"
+  },
+  "license": "Apache-2.0",
+  "typings": "api.d.ts",
+  "main": "lib/src/api.js",
+  "files": [
+    "api.d.ts",
+    "lib/src/**/*.js",
+    "lib/src/**/*.js.map"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Chevrotain/chevrotain.git"
+  },
+  "scripts": {
+    "---------- CI FLOWS --------": "",
+    "build": "npm-run-all clean compile",
+    "test": "npm-run-all coverage",
+    "---------- DEV FLOWS --------": "",
+    "update-snapshots": "node ./scripts/update-snapshots.js",
+    "---------- BUILD STEPS --------": "",
+    "clean": "shx rm -rf dev",
+    "compile:watch": "tsc -w",
+    "compile": "tsc",
+    "coverage": "nyc mocha"
+  },
+  "dependencies": {
+    "chevrotain": "9.0.1",
```

Comment:
I think that if I extract the GASTVisitor to a separate package we can avoid this dependency to Chevrotain

Reply:
That would work at the cost of complexity.

Was the main cause for this so that the generator api could be exposed in the api documentation?  I feel a better solution would be to tweak the documentation scripts than the production code.  Rather than the other way round.